app/serializers/api/current_order_serializer.rb

```diff
@@ -12,6 +12,10 @@ def payment_method_id
     object.payments.first.andand.payment_method_id
   end
 
+  def shipping_method_id
+    object.shipping_method.id
```

Comment:
I'm might have seen it and I don't remember but, did we add a `#shipping_method` that uses `shipments` under the hood?

Reply:
object.shipping_method takes you to the orderShippingMethod concern that uses order.shipments.